dev/docker-compose.yaml

```diff
@@ -214,7 +214,7 @@ services:
       REACT_APP_CLIENT_ID: 'summary_openid_client'
       REACT_APP_DASHBOARD_URL: 'https://femr.${BASE_DOMAIN:-localtest.me}'
       REACT_APP_PROJECT_ID: 'DCW'
-      REACT_APP_QUESTIONNAIRES: 'minicog,gad7,phq9'
+      REACT_APP_QUESTIONNAIRES: minicog,ecog12,cp-ecog,adl_iadl,gad7,gds,c-idas,behav5,slums,phq9
```

Comment:
@ivan-c I think in DCW, **phq9** is not administered.

Reply:
Agreed, @ivan-c  please exclude phq9

Reply:
Done!